include/stdexec/execution.hpp

```diff
@@ -1243,6 +1244,30 @@ namespace stdexec {
   /////////////////////////////////////////////////////////////////////////////
   // __connect_awaitable_
   namespace __connect_awaitable_ {
+    inline constexpr auto __co_call =
+      []<class _Fun, class... _Ts>(_Fun __fun, _Ts&&... __as) noexcept {
```

Comment:
Why a lambda instead of just a normal function?

Normal functions should give better info in stack-traces, diagnostics, etc. - naming `__co_call` instead of "::$lambda 25::operator()".